ansible/vagrant-playbook.yml

```diff
@@ -7,11 +7,13 @@
     ansible_python_interpreter: auto
   roles:
     # This role manage vagrant specificity
-    - role: vagrant
-    - role: updates
-    - role: svc_docker
-    - role: svc_traefik
-    - role: svc_minio
-    - role: svc_coredns
-    - role: svc_vault
-    - role: deerhide_users
+    # - role: vagrant
+    # - role: updates
+    # - role: svc_docker
+    # - role: svc_traefik
+    # - role: svc_minio
+    # - role: svc_coredns
+    - role: svc_prometheus
+    - role: svc_node_exporter
+    # - role: svc_vault
+    # - role: deerhide_users
```

Comment:
Is this intended? If not, please reenable these roles before it goes to main.

Reply:
Done